Formula/kopia.rb

```diff
@@ -0,0 +1,42 @@
+class Kopia < Formula
+  desc "Fast and secure open-source backup"
+  homepage "https://kopia.io"
+  url "https://github.com/kopia/kopia.git",
+      tag:      "v0.10.4",
+      revision: "3447f1865f0b0536653aa6e9c07896ce73c3e600"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/kopia/kopia/repo.BuildInfo=#{Utils.git_head}
+      -X github.com/kopia/kopia/repo.BuildVersion=#{version}
+    ]
```

Comment:
```suggestion
    ldflags = %W[
      -s -w
      -X github.com/kopia/kopia/repo.BuildInfo=#{Utils.git_head}
      -X github.com/kopia/kopia/repo.BuildVersion=#{version}
    ].join(" ")
```
Should we also set `github.com/kopia/kopia/repo.BuildGitHubRepo`? Otherwise the `from:` is blank in the version output.

Reply:
The `join` is not needed anymore, actually. `std_go_args` handles `Array` arguments now.

Reply:
kept as-is, without join